src/wp-admin/css/common.css

```diff
@@ -3591,6 +3583,15 @@ img {
 	display: none;
 }
 
+.control-section {
+	border: 1px solid #c3c4c7;
+	box-shadow: 0 1px 1px rgba(0, 0, 0, 0.04);
+}
+
+.control-section:focus-within {
+	border: 2px solid #3582c4;
```

Comment:
Having a 2px border here moves the section slightly when it becomes active. Would it make more sense to use `outline` here instead of the border? That doesn't move the element, and it is still highlighted.

Reply:
Excellent point! Now done.